packages/joy-actors/src/AvailableRoles/index.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react'
+import { BareProps } from '@polkadot/ui-app/types';
+import { ComponentProps } from '../props';
+import { Role, RoleParameters } from  '../types';
+import { Option, u32, Balance, BlockNumber } from '@polkadot/types';
+import { withCalls } from '@polkadot/ui-api/index';
+import { Table } from 'semantic-ui-react';
+import Section from '@polkadot/joy-utils/Section';
+
+import BN from 'bn.js';
+
+type Props = BareProps & ComponentProps;
+
+export default class AvailableRoles extends React.PureComponent<Props> {
+    render() {
+        return (
+            <div> {this.props.roles.map((role) => <div key={role.toString()}><RoleDisplay role={role}></RoleDisplay></div>) } </div>
```

Comment:
Let me suggest a little formatting here (easier to read):
```suggestion
            <div>{this.props.roles.map((role) =>
                  <div key={role.toString()}><RoleDisplay role={role} /></div>
            )}</div>
```